examples/plugins/README.md

```diff
@@ -9,3 +9,8 @@ is useful if you want to send EVE output to custom destinations.
 
 A minimal capture plugin that can be used as a template, but also used
 for testing capture plugin loading and registration in CI.
+
+## altemplate
+
+An app-layer template plugin with logging and detection.
+Most code copied from rust/src/applayertemplate
```

Comment:
nit: 
```suggestion
Most code copied from rust/src/applayertemplate.
```

Reply:
Would make it fell like the dot is part of the path :-/